stable/traefik/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: traefik
-version: 1.30.0
```

Comment:
Another PR has already been aproved and lgtm'ed that will bring version up to 1.31.0. So this will need to become 1.31.1 to be mergeable.